### PR TITLE
Fix ruff isort integration

### PR DESCRIPTION
**Pull request by samet-akcay**
# Description

- Fix isort configurations in `ruff`

## Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor (non-breaking change which refactors the code base)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the [pre-commit style and check guidelines](https://github.com/openvinotoolkit/anomalib/blob/main/CONTRIBUTING.md) of this project.
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] I have added a summary of my changes to the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) (not for minor changes, docs and tests).

### CODE REVIEW

**Review by djdameln — APPROVED** [review]

File: tests/pre_merge/utils/loggers/test_get_logger.py [hunk 1]
```diff
@@ -9,8 +9,6 @@
 from omegaconf import OmegaConf
 
 try:
```
Comment (ashwinvaidya17) [inline tests/pre_merge/utils/loggers/test_get_logger.py:11]: Why is this removed? If we don't try importing wandb then this try statement will always pass

**Review by ashwinvaidya17 — CHANGES_REQUESTED** [review]
Thanks. I have a few comments

File: tests/pre_merge/utils/loggers/test_get_logger.py (hunk 1 above)
Comment (samet-akcay) [reply tests/pre_merge/utils/loggers/test_get_logger.py:11]: it's an autofix by `ruff`. It complains that the import is not used.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -40,6 +40,7 @@ repos:
     hooks:
       - id: nbqa-black
       - id: nbqa-ruff
+        args: ["--ignore=I001"]
```
[the remark this answers is not in the document]
Comment (samet-akcay) [reply .pre-commit-config.yaml:45]: Done.

File: tests/pre_merge/utils/loggers/test_get_logger.py (hunk 1 above)
Comment (samet-akcay) [reply tests/pre_merge/utils/loggers/test_get_logger.py:11]: Ignoring that line now

**Review by samet-akcay — COMMENTED** [review]
@ashwinvaidya17, I've addressed your comments.

**Review by ashwinvaidya17 — APPROVED** [review]
Thanks
